soc/xtensa/intel_adsp/tools/cavstool.py

```diff
@@ -343,7 +537,10 @@ def ipc_command(data, ext_data):
         dsp.HIPCIDR = (1<<31) | ext_data
 
 async def main():
-    global hda, sd, dsp, hda_ostream_id
+    log.debug("Removing old hugetables")
+    runx("rm -rf /dev/hugepages/cavs_*")
```

Comment:
What's this for?  The old scheme would detect and re-use the file if it was present.

Reply:
I wanted fresh and clean, every run. Which I didn't get without it. One of the many things I ended up falling into a rabbit hole on while debugging, probably removable now